couchbase_cluster_admin/cluster.py

```diff
@@ -425,3 +436,115 @@ def set_autofailover(self, settings: dict):
         )
         if resp.status_code != 200:
             raise Exception(f"Failed to set auto failover settings: {resp.text}")
+
+    def delete_node_alternate_address(self):
+        url = f"{self.baseurl}/node/controller/setupAlternateAddresses/external"
+        resp = self.http_request(
+            url,
+            method="DELETE",
+        )
+        if resp.status_code != 200:
+            raise DeleteAlternateAddressException(resp.text)
+
+    def set_node_alternate_address(self, hostname: dict):
+        url = f"{self.baseurl}/node/controller/setupAlternateAddresses/external"
+        resp = self.http_request(
+            url,
+            method="PUT",
+            data={"hostname": hostname},
+        )
+        if resp.status_code != 200:
+            raise SetAlternateAddressException(resp.text)
+
+    def set_gsi_settings(self, gsi_settings: dict):
+        """
+        https://docs.couchbase.com/server/current/rest-api/post-settings-indexes.html
+        """
+        url = f"{self.baseurl}/settings/indexes"
+        resp = self.http_request(
+            url,
+            method="POST",
+            data=gsi_settings,
+        )
+        if resp.status_code != 200:
+            raise SetGsiSettingsExceptin(resp.text)
```

Comment:
```suggestion
            raise SetGsiSettingsException(resp.text)
```

Reply:
Fixed in 8acf35c34a979ab1f582e04ba3ce7d0c91a62352